pkg/solidity-utils/test/BalancerErrors.test.ts

```diff
@@ -17,4 +17,9 @@ describe('BalancerErrors', function () {
   it('translates the error code to its corresponding string if existent', async () => {
     await expect(errors.fail(102)).to.be.revertedWith('UNSORTED_TOKENS');
   });
+
+  it('encodes the prefix as expected', async () => {
+    // GYR = 0x475952
+    await expect(errors.failWithPrefix(123, '0x475952')).to.be.revertedWith('GYR#123');
```

Comment:
Can you also update the test above so we're looking for the entire `BAL#123` string?

Reply:
Done!